app/components/embed/pdf_component.html.erb

```diff
@@ -3,7 +3,7 @@
   <% # removing 5 pixels from the body height to account for being embedded %>
   <div class='sul-embed-body sul-embed-pdf'>
     <object data="<%= viewer.pdf_files.first %>" type="application/pdf" style="height: 100vh; width: 100%">
-      <p>Your browser does not support viewing PDFs.  Please <a href="<%= viewer.pdf_files.first %>">download the file</a> to view it.</p>
+      <p>Your browser does not support viewing PDFs.  Please <a target="_blank" href="<%= viewer.pdf_files.first %>">download the file</a> to view it.</p>
     </object>
```

Comment:
if the user needs to auth to view the PDF, this at least sends them to a new window